t/scripts/run_timeout.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""run command with a timeout, timeout as a float is first argument, rest are command"""
+import sys
+import signal
+import subprocess as s
+
+if len(sys.argv) < 3:
+    print("insufficient arguments passed to run_timeout")
```

Comment:
Nitpick.  Feel free to ignore. I only mention it because I think sharness splits stdout and stderr into separate places:

```suggestion
    print("insufficient arguments passed to run_timeout", file=sys.stderr)
```

Reply:
PS - Nevermind, stdout is actually what shows up in the log is you set `verbose=t`.  So what you have is totally correct.  Sorry for the noise.

```
❯ cat t8000-stdout-err.log                                                                                                             0:06:24 ()
testing stdout
expecting success: 
        echo "testing stdout" && true

testing stdout
ok 1 - stdout
PASS: t8000-stdout-err.t 1 - stdout

expecting success: 
        echo "testing stdout" 1>&2 && true

ok 2 - stderr
PASS: t8000-stdout-err.t 2 - stderr

# passed all 2 test(s)
1..2
```